internal/kafka/internal/services/clusters.go

```diff
@@ -22,6 +23,8 @@ import (
 	apiErrors "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
 )
 
+var kafkaStatusesThatNoLongerConsumesResourcesInTheDataPlane = []string{constants.KafkaRequestStatusDeleting.String()}
```

Comment:
minor nitpicking: it should be `kafkaStatusesThatNoLongerConsumeResourcesInTheDataPlane` (Consume instead of Consumes)

Reply:
Thanks!

Reply:
Updated :-)